docs/design/autoware-interfaces/components/map.md

```diff
@@ -0,0 +1,37 @@
+# Map
+
+![Node diagram](./images/Map-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+Autoware relies on high-definition point cloud maps and vector maps of the driving environment to perform various tasks. Before launch Autoware, you need to load the pre-created map files.
+
+How to create maps reference [Creating maps for Autoware](../../../how-to-guides/creating-maps-for-autoware.md).
+
+## Outputs
+
+### Point cloud map
+
+Loads pointcloud file and publishes the maps to the other Autoware nodes in various configurations. Currently, it supports the following types:
+
+- Raw pointcloud map (sensor_msgs/msg/PointCloud2)
+- Downsampled pointcloud map (sensor_msgs/msg/PointCloud2)
```

Comment:
I think downsampled maps are mainly for visualization so you might want to mention that in the documentation

Reply:
The pointcloud_map_loader node  publish the original point cloud and downsampled point cloud